src/main/java/com/elytradev/concrete/inventory/IObservable.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This is probably the one part of the diff I'm happiest with, having a dedicated Observable interface is a good call.